pkgs/applications/networking/juju/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, pkgs }:
+
+pkgs.buildGoModule rec {
+  pname = "juju";
+  version = "2.8.7";
+
+  # Disable tests because it attempts to use a mongodb instance
+  doCheck = false;
```

Comment:
Normally this goes between vendorSha256 and meta.

Reply:
Moved this.